recipes/fonduer/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "fonduer" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 499644f1648832cddf271416bb9c0cad7baa2300bb38d735892a410d8cf8e71b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - beautifulsoup4 ==4.7.1
+    - editdistance >=0.5.2
+    - emmental >=0.0.6
+    - ipython
+    - lxml >=4.2.5
+    - numpy >=1.11
+    - pyyaml >=5.1
+    - scipy >=1.1.0
+    - snorkel >=0.9.5
+    - spacy >=2.1.3
+    - sqlalchemy[postgresql] >=1.3.7
```

Comment:
conda doesn't use feature selection like this.  You may need to have another dependency

Reply:
No problem!  Yeah, you have to be careful when using taking dependcies from other package managers sometimes